hasura/migrations/default/1659467509391_set_fk_public_distributions_created_by/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."distributions"
+  add constraint "distributions_created_by_fkey"
+  foreign key ("created_by")
+  references "public"."profiles"
+  ("id") on update restrict on delete restrict;
```

Comment:
I wasn't sure if these should be `restrict` or something else... restrict seems to make sense since we're doing soft deletes and hard deletes would mess with our data.

Reply:
yeah restrict is good enough